Changefeed/ChangefeedSample/ChangefeedSample.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AzureFunctionsVersion>v3</AzureFunctionsVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Azure.Storage.Blobs.ChangeFeed" Version="12.0.0-dev.20200731.1" />
```

Comment:
I think we confirmed that the fixes didn't work right? Should we revert this back to depend on the currently released preview version for now?